qa/tasks/vstart_runner.py

```diff
@@ -381,7 +381,12 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
                     args[0]
                 ))
 
-        log.info("Running {0}".format(args))
+        logmsg = '>'
+        for arg in args:
+            # we need to convert to string in case of args like Raw('&&')
+            logmsg += ' ' + (str(arg.value) if isinstance(arg, Raw) else arg)
```

Comment:
Why not giving a try to [list comprehensions](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions):
```suggestion
        logmsg = '> {}'.format(' '.join(str(arg.value) if isinstance(arg, Raw) else arg for arg in args)
```

Reply:
Yes, `str.join` and list comprehension should make it much more cleaner.

Fixed. Thanks!